### DISCUSSION TITLE
Some shortcuts missing icons

### DISCUSSION THREAD

**Discussion by Fraggle121** [moraroy/NonSteamLaunchers-On-Steam-Deck#328] [Q&A]
Ive tried on both deck version and script version and i just cannot  get all the non steam launchers shortcuts to appear. it does work and create them for epic, gog and amazon, but doesnt create them for battlenet, EA, ubisoft and rockstar, and ideas. it did work when i first set them up but had to reinstall to fix the battlent update. ive tried doing fresh,  seperate ID, all in same ID, im at a loss

**Comment by moraroy** [comment]
@Fraggle121 sorry i didnt see this, is this still an issue for you? also. please uplaod your log
